crawler.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+FileUtils.rm_rf('data')
+
+t1 = Time.now
+puts "Started at: #{t1}"
```

Comment:
It's already done the relaton-w3c. I don't think we need to put start/stop time here.

Reply:
Done